backpack/extensions/secondorder/hbp/custom_module.py

```diff
@@ -0,0 +1,10 @@
+from backpack.core.derivatives.sum_module import SumModuleDerivatives
+from backpack.extensions.secondorder.hbp.hbpbase import HBPBaseModule
+
+
+class HBPSumModule(HBPBaseModule):
+    """KFAC extension for SumModule."""
+
+    def __init__(self):
+        """Initialization."""
+        super().__init__(derivatives=SumModuleDerivatives())
```

Comment:
Can you add this file to `fully_documented.txt` such that the docstring CI will be triggered for it?

Reply:
Fixed in commit
"Updated docstrings to fix doc error"

Reply:
Nice. Closing.